pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -324,6 +324,11 @@ func (m *MultishareOpsManager) runEligibleInstanceCheck(ctx context.Context, ins
 	var readyEligibleInstances []*file.MultishareInstance
 	nonReadyInstanceCount := 0
 	for _, instance := range instances {
+		if instance.State != "READY" {
```

Comment:
we should consider CREATING as a non ready instance too

Reply:
what if the instance is in the process of restarting? do we need any logic to handle this case specifically or "not ready" can work for this case too?